src/index.js

```diff
@@ -165,6 +172,18 @@ export default function undoable(reducer, rawConfig = {}) {
     default:
       res = reducer(state && state.present, action);
 
+      if (Array.isArray(config.initActions)) {
```

Comment:
That's why I used plural, but yeah, I will add the check when building config and wrap the value into array.